test_pytest_html.py

```diff
@@ -261,6 +255,31 @@ def pytest_runtest_makereport(item, call):
             href)
         assert link in html
 
+    def test_extra_text_separeted(self, testdir):
+        content = str(random.random())
+        testdir.makeconftest("""
+            import pytest
+            @pytest.mark.hookwrapper
+            def pytest_runtest_makereport(item, call):
+                outcome = yield
+                report = outcome.get_result()
+                if report.when == 'call':
+                    from pytest_html import extras
+                    report.extra = [extras.text('{0}')]
+        """.format(content))
+        testdir.makepyfile('def test_pass(): pass')
+        result, html = run(testdir)
+        assert result.ret == 0
+        hash_key = ('test_extra_text_separeted.py::'
+                    'test_pass01').encode('utf-8')
+        hash_generator = hashlib.md5()
+        hash_generator.update(hash_key)
+        href = '{0}/{1}'.format('assets', '{0}.txt'.
+                               format(hash_generator.hexdigest()))
+        link = '<a class="text" href="{0}" target="_blank">Text</a>'.format(
+            href)
+        assert link in html
+
```

Comment:
@davehunt I intend to do one test of text to rerun, and two new ones to JSON (separated "normal" and separated  with rerun (and now I realized a typo in "separeted")